app/main.py

```diff
@@ -1,2 +1,72 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+from fractions import Fraction
+
+CAPACITY = 8
+INCREASE_MULTIPLIER = 2
+LOAD_FACTOR = Fraction(2, 3)
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8) -> None:
```

Comment:
capacity shouldn't be passed as parameter - it cannot be set from outside of Dict